lib/CampMinder/SignedRequestFactory.rb

```diff
@@ -17,33 +17,16 @@ def is_valid_request?(signed_request)
   end
 
   def get_payload(signed_payload)
-    Base64.decode64(prepare_decode_base64(signed_payload.split('.').last))
+    Base64.urlsafe_decode64(signed_payload.split('.').last)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.